### ISSUE TITLE
Dreary doesn't work

### ISSUE THREAD

**Issue by Tobiasgrn** [morpheusthewhite/spicetify-themes#513]
I have used the command spicetify config sidebar_config 1 and then spicetify apply as intended but nothing happend to the sidebar it is still just standard but the theme works perfectly fine.

**Comment by CharlieS1103** [comment]
Yeah sorry about that, the latest update screwed a few things up with sidebar-config(which by the way make sure is set to 1), I'll try and fix that when I find time but am currently on vacation.

**Comment by CharlieS1103** [comment]
Just to add on, there's a chance that if you had sidebar config disabled in the config, that if you just enable it ,Dreary will work.

**Comment by CharlieS1103** [comment]
As it turns out, the latest update of Spotify (the one that broke spicetify) happened to break everything else too, unless spicetify is able to be fixed the only way for dreary to work as intended is to downgrade, for now, I'll look into seeing if I can get it to work on this version but it may be difficult.

**Comment by Tobiasgrn** [comment]
> As it turns out, the latest update of Spotify (the one that broke spicetify) happened to break everything else too, unless spicetify is able to be fixed the only way for dreary to work as intended is to downgrade, for now, I'll look into seeing if I can get it to work on this version but it may be difficult.

Thanks for trying tho. :-)
